bin/install-qa-check.d/60pkgconfig

```diff
@@ -104,7 +104,7 @@ pkgconfig_check() {
 		local -A bad_files
 		for f in "${files[@]}" ; do
 			local file_version=$(pkg-config --modversion "${f}")
-			if [[ ${QA_PKGCONFIG_VERSION} == ${file_version} ]] ; then
+			if ver_test ${QA_PKGCONFIG_VERSION} -eq ${file_version}; then
```

Comment:
I am not entirely sure if we can use `ver_test` here. A quick test showed that it works, but it may be fragile. That is, it only worked because the ebuild had the right EAPI.

Furthermore, `ver_test` requires all arguments to conform to PMS version strings. This may not be the case for `$file_version`. Not sure if this is then undefined behavior that could cause havoc?